test/infrastructure/docker/exp/api/v1beta1/dockermachinepool_types.go

```diff
@@ -114,6 +114,7 @@ type DockerMachinePoolInstanceStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Cluster",type="string",JSONPath=".metadata.labels['machine\\.cluster\\.x-k8s\\.io/cluster-name']",description="Cluster"
```

Comment:
```suggestion
```
We don't set a cluster-name label on this resource

Reply:
Fine to remove the column for now but I consider this a bug, let's open an issue

Reply:
Agree

Reply:
Done > https://github.com/kubernetes-sigs/cluster-api/issues/5340